pkg/trace/config/apply.go

```diff
@@ -369,6 +435,7 @@ func (c *AgentConfig) applyDatadogConfig() error {
 			}
 		}
 	}
+
```

Comment:
Please remove it. Check point 3 here https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines